### PR TITLE
Add a home page to kroxylicious.io

### PR DESCRIPTION
**Pull request by racheljpg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+
+## Organisation
+
+This effort is currently being bootstrapped, take a look at the following resources to learn more about the current state of discussions, thinking and first PoC:
+
+* [Our GitHub Org](https://github.com/kroxylicious): Kroxylicious Organisation
+* [kproxy](https://github.com/kroxylicous/kproxy): A basic PoC for a Kafka protocol proxy based on Netty
+* [Design](https://github.com/kroxylicious/design): Design thoughts & concepts
+* [Discussions](https://github.com/kroxylicous/design/discussions): Ideas and other discussions
+* [Resources](useful-resources.asciidoc): A collection of useful resources
```
Comment (katheris) [inline README.md:18]:
```suggestion
* [Resources](https://github.com/kroxylicious/design/blob/main/useful-resources.asciidoc): A collection of useful resources
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
-# kroxylicious.io
-A repository to host the website for Kroxylicious.
+# Kroxylicous
+
+The goal of the Kroxylicous effort is to explore the creation of a Kafka protocol proxy,
+addressing use cases such as multi-tenancy, schema validation, or encryption.
+
+## Design Goals
```
Comment (katheris) [inline README.md:6]: We can probably omit this

File: README.md [hunk 3]
```diff
@@ -1,2 +1,22 @@
...
+
+This effort is currently being bootstrapped, take a look at the following resources to learn more about the current state of discussions, thinking and first PoC:
+
+* [Our GitHub Org](https://github.com/kroxylicious): Kroxylicious Organisation
+* [kproxy](https://github.com/kroxylicous/kproxy): A basic PoC for a Kafka protocol proxy based on Netty
+* [Design](https://github.com/kroxylicious/design): Design thoughts & concepts
+* [Discussions](https://github.com/kroxylicous/design/discussions): Ideas and other discussions
+* [Resources](useful-resources.asciidoc): A collection of useful resources
+
+## License
```
Comment (katheris) [inline README.md:20]: Should we omit this and instead add a link in the list above that links to https://github.com/kroxylicious/kproxy/blob/main/LICENSE ?

**Review by katheris — COMMENTED** [review]
Generally looks good to me, just a couple of suggested changes

**Comment by racheljpg** [comment]
Thanks for the review, @katheris! I've added your suggestions

**Review by katheris — APPROVED** [review]
LGTM

File: README.md [hunk 4]
```diff
@@ -1,2 +1,15 @@
-# kroxylicious.io
-A repository to host the website for Kroxylicious.
+# Kroxylicous
```
Comment (tombentley) [inline README.md:1]:
```suggestion
# Kroxylicious
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,15 @@
...
+
+The goal of the Kroxylicous effort is to explore the creation of a Kafka protocol proxy,
+addressing use cases such as multi-tenancy, schema validation, or encryption.
+
+## Organisation
+
+This effort is currently being bootstrapped, take a look at the following resources to learn more about the current state of discussions, thinking and first PoC:
```
[the remark this answers is not in the document]
Comment (katheris) [reply README.md:8]: I like this suggested longer summary, it gives more background to the project and it's initial design aims

**Review by k-wall — APPROVED** [review]
With the edits suggested +1

**Pull request merged**